First commit of DemoX v2 (2024) course

jswope00 opened the pull request:
This is the DemoX v2 course in full. I still have to write an about page which I'll add as another pull request.

jswope00 committed:
feat: First commit of DemoX v2 (2024) course

jswope00 committed:
First commit of DemoX v2 (2024) course

openedx-webhooks commented:
Thanks for the pull request, @jswope00! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.




<!-- comment:external_pr -->
<!-- data: eyJkcmFmdCI6IGZhbHNlfQ== -->

jswope00 committed:
feat: Add the demo library which can be used in conjunction with the DemoX v2 course.

jswope00 committed:
Add the demo library which can be used in conjunction with the DemoX v2 course.

jswope00 committed:
feat: Commit the library as a zipped .tar.gz file

jswope00 committed:
Commit the library as a zipped .tar.gz file

jswope00 committed:
feat: Commit the course as a .tar.gz file

jswope00 committed:
Commit the course as a .tar.gz file

jswope00 committed:
Update demo-course README.rst

Update the readme to explain the purpose of the demo-course directory and tell users to use the instructions in the main README for importing

jswope00 committed:
Update Library README.rst

Update the Library's Readme file to explain what the directory contains and how to import the library into a course.

jswope00 committed:
build: Merge remote-tracking branch 'origin/main'

jswope00 committed:
Merge remote-tracking branch 'origin/main'

openedx-webhooks commented:
@jswope00 Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/openedx-demo-course-new&entry.867055334=https://github.com/openedx/openedx-demo-course-new/pull/1&entry.1484655318=https://github.com/jswope00&entry.752974735=2023-12-18+20:37&entry.1917517419=2023-12-20+21:38&entry.2133058324=No) so we can improve your experience in the future.

<!-- comment:end_survey -->
